monkey/monkey_island/cc/server_utils/encryption/field_encryptors/string_list_encryptor.py

```diff
@@ -6,9 +6,9 @@
 
 class StringListEncryptor(IFieldEncryptor):
```

Comment:
I don't understand why we need this encryptor at all. It doesn't encrypt a list, it encrypts each string in a list. And we already have a string encryptor. Python also has methods like `map` to apply a function to each element of the list.

Reply:
Once we fully refactor `ReportService` or add a repository for reports, we don't.

If I just remove the passwords from the report as per https://github.com/guardicore/monkey/pull/2088#discussion_r920864626, then we don't need to wait any longer to remove it.